src/components/Articles.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState, useEffect } from "react";
+import { getAllArticles } from "../utils/api";
+
+const AllArticles = () => {
+  const [isLoading, setIsLoading] = useState(true);
+  const [articles, setArticles] = useState([]);
+  const [error, setError] = useState(null);
+
+  useEffect(() => {
+    const fetchArticles = () => {
+      getAllArticles()
+        .then((data) => {
+          setArticles(data);
+          setIsLoading(false);
+        })
+        .catch((error) => {
+          setError(error);
+          setIsLoading(false);
+        });
+    };
+    fetchArticles();
+  }, []);
+
+  return (
+    <div className="articles-div">
+      <h2>Click on desired article for details</h2>
+      {isLoading ? (
+        <p>Loading...</p>
+      ) : error ? (
+        <p>Error: {error.message}</p>
+      ) : (
+        <ul>
+          {articles.map((article) => (
```

Comment:
You could/should separate this into it's own component `ArticleCard.jsx` or something like that